src/ResourceManager/ApiManagement/Commands.ApiManagement.ServiceManagement/Commands.ApiManagement.ServiceManagement.csproj

```diff
@@ -119,10 +119,7 @@
     </Reference>
```

Comment:
rather than reverting them explicitly can't you just check out the previous versions of these files?